BackToBasicsUIKit/BackToBasicsUIKit.xcodeproj/project.pbxproj

```diff
@@ -90,6 +106,7 @@
 		EF8F5BF3221328F50017C9FF /* BackToBasicsUIKit */ = {
 			isa = PBXGroup;
 			children = (
+				F6F83AFC222F544500CFDF42 /* Audio */,
```

Comment:
I would keep this folder inside of your folder.

Reply:
Relocated